### DISCUSSION TITLE
Shared folders on Sonoma guests?

### DISCUSSION THREAD

**Discussion by keeely** [utmapp/UTM#5978] [macOS]
Is there some trick to getting shared folders working from Sonoma guests?
UniversalMac_13.5.2_22G91_Restore.ipsw (Ventura)- working as expected.  "My Shared Files" appears in finder, functions as expected when I add a share in the VM settings.
UniversalMac_14.2_23C64_Restore.ipsw (Sonoma) - not working.  There is no My Shared files in finder, even though I added a share.

My host is Sonoma.  Running UTM 4.4.5. As far as I can see the two VMs are identical except for the IPSW they are based on. 

I wonder if a few other people may hit this now that (I think) Sonoma is going to be the default IPSW.

**Comment by jamesldt** [comment]
Have the same problem - can't figure out how to find the UTM shared folder.

Using same versions of Sonoma for host as well as guest. File sharing works fine (i.e. "Connect to Server") in both directions, for the shared folder as well as anything else on the local net or host or VM (have to use VM's IP to connect to it).  In the browser (Brave & Safari) I've tried many variations of localhost:9843 (https://localhost:9843, https://127.0.0.1:9843, https://192.168.7.151 [VM's DHCP assigned IP]) and some others I forget.  HOWEVER..., now that I think about it, I also have no Internet/web access on the VM Sonoma - even though everything on the VM's DHCP network settings and firewall setting looks good. So I guess the real first problem is why I've no browser access. I did try both types of network UTM allows a Mac VM (Bridged & Shared), but with no difference I could discern.

The "mount_virtiofs share /Users/me/Share" command returns "no tag named 'share' " (or something to that effect - i.e., no "share" tag has been established for the mount_virtiofs command to find).

Other network stuff seems to work fine in the VM (e.g., printers, file sharing).

UTM 4.4.4
MacOS Sonoma 14.3.1
MacBook Air (M2)
